README.md

```diff
@@ -97,15 +98,14 @@ include any necessary hooks in a file called `custom.py` for Python models or `c
 - `read_input_data(input_filename: str) -> Any`
   - `input_filename` is a data file, passed in the `--input` parameter
   - If used, this hook must return a non-None value; if it returns something other than a DF, you'll need to write your own score method.
-  - This hook can be used to customize data file reading, e.g: mode, encoding.
-  **drum** does not natively support
+  - This hook can be used to customize data file reading, e.g: mode, encoding, handle missing values.
 - `load_model(code_dir: str) -> Any`
   - `code_dir` is the directory where the model artifact and additional code are provided, which is passed in the `--code_dir` parameter
   - If used, this hook must return a non-None value
   - This hook can be used to load supported models if your model has multiple artifacts, or for loading models that
   **drum** does not natively support
 - `transform(data: DataFrame, model: Any) -> DataFrame`
-  - `data` is the dataframe given to **drum** to make predictions on
+  - `data` is the dataframe given to **drum** to make predictions on. Missing values are replaced with NaN in Python, with NA in R.
```

Comment:
I would state this as

> Missing values are indicated with `NaN` in Python and `NA` in R, unless otherwise overridden by the `read_input_data` hook.